content/pl/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Cloud Controller Manager
+id: cloud-controller-manager
+date: 2018-04-12
+full_link: /docs/concepts/architecture/cloud-controller/
+short_description: >
+  Element warstwy sterowania, który integruje Kubernetesa z zewnętrznymi usługami chmurowymi.
+aka: 
+tags:
+- core-object
+- architecture
+- operation
+---
+Element składowy {{< glossary_tooltip text="warstwy sterowania" term_id="control-plane" >}} Kubernetesa,
+który zawiera logikę sterowania specyficzną dla usług chmurowych. Cloud controller manager umożliwia
```

Comment:
```suggestion
który zawiera logikę odpowiedzialną za zarządzanie dedykowanymi usługami chmur obliczeniowych. Cloud controller manager umożliwia
```

Reply:
This has been reworded in the next commit.